docs/antora.yml

```diff
@@ -1,5 +1,5 @@
 name: quarkus-openapi-generator
-title: Quarkus - Openapi Generator
+title: '- Openapi Generator'
```

Comment:
@gastaldi I just noticed those single quotes. Is that right? The other PRs don't have them.

Reply:
No, this is wrong, it was created by my script. Let me fix it